### PR TITLE
Add browser build

### LINKED ISSUE #187: Build for browsers
We will use globalstorage in browser in next few weeks, so first we need to build dependencies:
- [x] msdf - https://github.com/metarhia/mdsf/issues/26 or make msdf optional
- [x] common - https://github.com/metarhia/common/issues/183
- [x] metasync - https://github.com/metarhia/metasync/issues/279
- [ ] make mkdirp optional

### PR DESCRIPTION
**Pull request by nechaido**
Closes #187.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,24 +20,38 @@
...
   "devDependencies": {
     "eslint": "^5.5.0",
     "eslint-config-metarhia": "^4.0.0",
     "eslint-plugin-import": "^2.14.0",
     "metaschema": "^0.0.21",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
```
Comment (aqrln) [inline package.json]: Either this must be a `dependency`, or `plugin-transform-runtime` must not be used at all.

**Comment by nechaido** [comment]
@aqrln ping.

File: package.json [hunk 2]
```diff
@@ -20,24 +20,38 @@
...
-    "lint": "eslint ."
+    "lint": "eslint . --ignore-path .gitignore",
+    "build": "babel gs.js -d dist && babel lib -d dist/lib --ignore lib/fs.provider.js,lib/fs.cursor.js",
+    "prepublish": "npm run -s build"
   },
   "engines": {
     "node": ">=6.0.0"
   },
   "dependencies": {
+    "@babel/runtime": "^7.0.0",
```
Comment (aqrln) [inline package.json]: Keep in mind that it makes `@babel/runtime` a dependency of the package, and it won't be used much of the time (i.e., in Node.js). Can you please compare the build size with and without `@babel/plugin-transform-runtime`?

Comment (nechaido) [reply package.json]: It's 64K vs 68K.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/globalstorage/commit/681094ce99f9fe475bff7fbdac13781e3d29594b.

**Pull request closed without merging**